### ISSUE TITLE
Limit the number of active promotions and demotions

### ISSUE THREAD

**Issue by karknu** [input-output-hk/ouroboros-network#4600]
Limit the number of active promotions and demotions performed by the p2p governor.

**Comment by dcoutts** [comment]
That is, limit the number of _concurrent_ promotions/demotions.

**Comment by karknu** [comment]
> That is, limit the number of _concurrent_ promotions/demotions.

Yes, given that the governor has targets for "active peers" I shouldn't have used the world "active".
